roles/openshift_master/tasks/main.yml

```diff
@@ -46,7 +46,9 @@
       extension_scripts: "{{ openshift_master_extension_scripts | default(None) }}"
       extension_stylesheets: "{{ openshift_master_extension_stylesheets | default(None) }}"
       extensions: "{{ openshift_master_extensions | default(None) }}"
-      oauth_template: "{{ openshift_master_oauth_template | default(None) }}"
+      oauth_template: "{{ openshift_master_oauth_template | default(None) }}" # deprecated
+      oauth_templates: "{{ openshift_master_oauth_templates | default(None) }}"
```

Comment:
Should these be set the same way to cover backwards/forwards compatibility for now?

We should probably mark the version the old options are deprecated in so that we know when they can be officially removed.